src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,47 @@
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from './SearchLink';
+import { getSearchWith } from '../utils/searchHelper';
+
 export const PeopleFilters = () => {
+  const centuries = ['16', '17', '18', '19', '20'];
+  const [searchParams, setSearchParams] = useSearchParams();
+  const sexFilter = searchParams.get('sex');
```

Comment:
Would be great to create an enum for search params to avoid errors connected with the typos in the future